### PR TITLE
chore(test): add multi-tenancy UI integration test cases

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Add multi-tenancy UI integration test cases:
1. delete tenant
2. create tenant from tenant landing page (for empty tenant state)
3. can not open create tenant modal when number of tenants reach the limit (max of 3 tenants)

Refactor:
Wrap some common user GUI operations as functions for reuse.

## Testing

N/A

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [x] integration tests
- [ ] docs

OR

- [ ] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/integration-tests/src/tests/ui-cloud/smoke.test.ts [hunk 1]
```diff
@@ -208,4 +162,78 @@ describe('smoke testing for cloud', () => {
...
+
+  it('can create two more tenant for new account', async () => {
+    await createNewTenant(page, 'tenant2');
+    await page.waitForTimeout(5000);
+    await createNewTenant(page, 'tenant3');
+    await page.waitForTimeout(5000);
+    expect(new URL(page.url()).pathname.endsWith('/get-started')).toBeTruthy();
+  });
+
+  it('can not open create tenant modal when reach the limit', async () => {
```
Comment (wangsijie) [inline packages/integration-tests/src/tests/ui-cloud/smoke.test.ts:226]: How can we know for sure that it failed?

**Review by wangsijie — APPROVED** [review]

File: packages/integration-tests/src/tests/ui-cloud/operations.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  await page.click('div[class$=continueActions] button:last-child');
+};
+
+export const onboardingFinish = async (page: Page) => {
+  // Wait for the sie config to load
+  await page.waitForTimeout(1000);
+
+  // Click the enter ac button
+  await page.click('div[class$=content] >button');
+
```
Comment (charIeszhao) [inline packages/integration-tests/src/tests/ui-cloud/operations.ts:42]: nit: SIE and AC should be in uppercase

File: packages/integration-tests/src/tests/ui-cloud/operations.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  await page.click(
+    'div[class$=ReactModalPortal] div[class*=card][class$=medium] div[class$=footer] button[type=submit]'
+  );
+};
+
+export const createNewTenant = async (page: Page, tenantName: string) => {
+  await page.waitForTimeout(500);
+  await openTenantDropdown(page);
+
+  await page.waitForTimeout(500);
```
Comment (charIeszhao) [inline packages/integration-tests/src/tests/ui-cloud/operations.ts:90]: No need to change timeout?

File: packages/integration-tests/src/tests/ui-cloud/smoke.test.ts [hunk 4]
```diff
@@ -51,16 +62,7 @@ describe('smoke testing for cloud', () => {
...
-    await expect(page).toClick('div[role=radio]:has(input[name=project][value=personal])');
-
-    // Select the deployment type option
-    await expect(page).toClick(
-      'div[role=radio]:has(input[name=deploymentType][value=open-source])'
-    );
-
-    // Click the next button
-    await expect(page).toClick('div[class$=actions] button:first-child');
+    await onboardingWelcome(page);
```
Comment (charIeszhao) [inline packages/integration-tests/src/tests/ui-cloud/smoke.test.ts:65]: Nit: function names should be verb-object phrases.

File: packages/integration-tests/src/tests/ui-cloud/smoke.test.ts [hunk 5]
```diff
@@ -208,4 +162,84 @@ describe('smoke testing for cloud', () => {
...
+  });
+
+  it('can create tenant from landing page', async () => {
+    const createTenantButton = await page.waitForSelector(
+      'div[class$=pageContainer] div[class$=placeholder] button[class$=button][type=button]'
+    );
+    await createTenantButton?.click();
+
+    await fillAndCreateTenant(page, 'tenant1');
+    await page.waitForTimeout(5000);
```
Comment (charIeszhao) [inline packages/integration-tests/src/tests/ui-cloud/smoke.test.ts:214]: Also here, do we really need to change timeout?

**Pull request closed without merging**